CHANGELOG.md

```diff
@@ -4,6 +4,14 @@
 
 - "You miss 100 percent of the chances you don't take. — Wayne Gretzky" — Michael Scott
 
+## 6.17.4
+
+- chore(deps): Bump `@sentry/webpack-plugin` from 1.18.3 to 1.18.4 (#4464)
+- fix(browser): Set severity level for events captured by the global error handler (#4460)
+- fix(nextjs): Export `BrowserTracing` integration directly (#4480)
+- fix(tracing): Export SpanStatus enum (#4478)
+- fix(vue): Property _isVue not defined in Vue3 (#4461)
+
```

Comment:
Realized we forgot to add contributers here, probably wanna add that before we merge.

Reply:
Oh, right. That's not in my muscle memory yet. Can do.